solc/CommandLineInterface.h

```diff
@@ -84,6 +85,7 @@ class CommandLineInterface
 	void printVersion();
 	void printLicense();
 	void compile();
+	void assemble();
```

Comment:
Naming it this way is confusing. We already have another `assemble()` method that's used for Yul.

```suggestion
	void assembleEVMAsm();
```
And I would rename the other one to `assembleYul()`.

Reply:
Ok

Reply:
Renamed `assemble(..)` to `assembleYul(..)` and `assemble()` to `assembleEvm()`.